src/testSetup/groovy/com/bmuschko/gradle/docker/TestUtils.groovy

```diff
@@ -0,0 +1,22 @@
+package com.bmuschko.gradle.docker
+
+class TestUtils {
+	/**
+	 * Escape backslashes in windows in docker files
+	 */
+	public static String escapeFilePath(File file) {
+		if (file.separatorChar == '\\') {
```

Comment:
Could you please fix indentation? I am not sure what you use but we use 4 spaces in this project.